app/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,272 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
To fix the compiler error about `tools`, you need to tweak one thing here (the translations editor is most likely completely unaware of this).

```suggestion
<resources xmlns:tools="http://schemas.android.com/tools">
```